spec/Section 5 -- Validation.md

```diff
@@ -675,16 +728,34 @@ fragment argOnRequiredArg on Dog {
 fragment argOnOptional on Dog {
   isHouseTrained(atOtherHomes: true) @include(if: true)
 }
+
+fragment withFragmentArg($command: DogCommand) on Dog {
+  doesKnowCommand(dogCommand: $command)
+}
+
+fragment usesFragmentArg on Dog {
+  ...withFragmentArg(command: DOWN)
+}
 ```
 
-the following is invalid since `command` is not defined on `DogCommand`.
+The following is invalid since `command` is not defined on
+`Dog.doesKnowCommand`.
 
 ```graphql counter-example
 fragment invalidArgName on Dog {
   doesKnowCommand(command: CLEAN_UP_HOUSE)
 }
 ```
 
+and this is also invalid as the variable `dogCommand` is not defined on fragment
```

Comment:
Oh no wait this is that weird thing where we're calling them variables like operation variables rather than arguments... In which case, we should reference "the variable `$dogCommand`"?

I think calling them arguments makes more sense here at least. We'll have to be careful with this.